src/Node/Render.php

```diff
@@ -29,7 +51,9 @@ public function __construct(Twig_Node_Expression $expr, Twig_Node_Expression $va
   }
 
   /**
-   * Adds the template arguments including the component (variant) context.
+   * Adds the component context variables to the Twig compiler string.
```

Comment:
The suggested PHPDoc makes sense for explaining the intended internals of the method here.

In addition, we should also consider to copy an explanation like this into the bird-level documentation, which currently seems to live on the main TwigFractal class.